homeassistant/components/onewire/strings.json

```diff
@@ -0,0 +1,30 @@
+{
+  "config": {
+    "abort": {},
+    "error": {
+      "connection_error": "Unable to connect to OWServer.",
+      "invalid_path": "Directory not found."
```

Comment:
Hi @Misiu 
I'm not sure what you mean by that.
I'm letting the user select the connection type (OWServer using host/port, OWFS by specifying a mount directory or SYSBUS)

Are you implying that I should hide SYSBUS if the DEFAULT_MOUNT_DIR (/sys/bus/w1/devices/) doesn't exist?

Reply:
I missed the error message about dtoverlay. I think this will be enough.
Don't hide that option. It's better to get an error that might help than to confuse the user by hiding the option.